src/core/wb.js

```diff
@@ -571,14 +571,29 @@ Modernizr.load( [
 					// Start initialization
 					wb.init( document, componentName, selector );
 
+					// Disable MathJax's context menu to more closely mimic native MathML implementations
+					window.MathJax = {
+						options: {
+							enableMenu: false
+						}
+					};
+
 					// Load the MathML dependency. Since the polyfill is only loaded
 					// when !Modernizr.mathml, we can skip the test here.
 					Modernizr.load( [ {
-						load: "timeout=500!https://cdn.jsdelivr.net/npm/mathjax@WET_BOEW_VERSION_MATHJAX/MathJax.js?config=Accessible",
+
+						// Load ES6 polyfill (for IE11) and MathJax from CDNs
+						load: [
+							"timeout=500!https://polyfill.io/v3/polyfill.min.js?features=es6",
+							"timeout=500!https://cdn.jsdelivr.net/npm/mathjax@WET_BOEW_VERSION_MATHJAX/es5/mml-chtml.js"
```

Comment:
Considering the polyfill is required to run the mml-chtml.js, ensure the polyfill is loaded and ready to be used before to load the mathJax.

Reply:
My latest changes bring it in prior to MathJax 3, although there aren't any extra checks in place atm to be completely certain that it's ready to rock and roll.

Reply:
Gonna mark this as resolved too since I don't think this is still relevant (due to ``setTimeout()``).